lib/strings.md

```diff
@@ -5,20 +5,26 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a String?
 
-Replace me with your response...
+A sequence of characters grouped together, enclosed by either single or double quotation marks.
 
 # What are some examples of information that would be Strings as opposed to some other data type?
 
-Replace me with your response...
+Text in any length - letters, words, sentences - would be treated as a string. 
+Numerals used in a non-mathematical way (as a digit) would be part of a string, but if intended to be used in calculations, would be a number, rather than a string.
 
 # What is one way, using Ruby, to retrieve the 6th character in a String like `"Ada Lovelace"`? How about the 8th character? What happens if you try to retrieve the value of the _99th_ character (Or any character that doesn't exist)?
 
-Replace me with your response...
+The code to retrieve the sixth character, then the eighth character, is as follows:
+a = "Ada Lovelace"
+puts a[6]
+puts a[8]
+
```

Comment:
I'm not following your comment. I am thinking of A as the 0th character - I'm not sure why I wouldn't?